### PR TITLE
update cli.md

### PR DESCRIPTION
**Pull request by RubenKelevra**
fixes https://github.com/ipfs/docs/issues/482

### CODE REVIEW

File: docs/reference/cli.md [hunk 1]
```diff
@@ -13,7 +13,7 @@ description: Command-line interace (CLI) reference materials for IPFS, the Inter
 
 This document is autogenerated from <a href='https://github.com/ipfs/docs/blob/master/scripts/cli.sh'>scripts/cli.sh</a>. For issues and support, check out the <a href='https://github.com/ipfs/docs'>docs repository</a> on GitHub.
 
-_Generated on 2019-08-30 18:40:23, from go-ipfs 0.4.22._
+*Generated on 2020-04-05 23:53:21, from go-ipfs 0.5.0-dev.*
```
Comment (hsanjuan) [inline docs/reference/cli.md:16]: We should do from 0.4.23 which is the current one.

Comment (RubenKelevra) [reply docs/reference/cli.md:16]: Why? Is this page automatically pushed to the website?

I thought your intention is to prepare the docs for the upcoming 0.5.0 release, that we just have to tag it and release the new documentations with the release.

Comment (hsanjuan) [reply docs/reference/cli.md:16]: Yes, on merge it publishes the website. I just want to update to the current stable release and once 0.5.0 happens we can update again.

**Comment by johnnymatthews** [comment]
Thanks for bringing this up @RubenKelevra. I just ran the script with Go-IPFS 0.5 installed. Can merge once all the checks have finished.

**Comment by RubenKelevra** [comment]
Cool :)

**Comment by johnnymatthews** [comment]
That last build error is because CI won't preview-deploy to IPFS from branches outside of the IPFS org. I'm gonna merge things into `master` and keep an eye on things just to make sure nothing breaks.

**Comment by johnnymatthews** [comment]
[Looks good](https://circleci.com/gh/ipfs/ipfs-docs-v2/826). Again, thanks for bringing this up @RubenKelevra.

**Pull request merged**